src/PrestaShopBundle/Resources/config/services/adapter/quick_access.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  _defaults:
+    public: false
+    bind:
+      $connection: '@doctrine.dbal.default_connection'
+      $dbPrefix: '%database_prefix%'
+
+  PrestaShop\PrestaShop\Core\QuickAccess\QuickAccessRepositoryInterface:
+    alias: PrestaShop\PrestaShop\Adapter\QuickAccess\Repository\QuickAccessRepository
+    public: true
+
+  PrestaShop\PrestaShop\Adapter\QuickAccess\Repository\QuickAccessRepository:
+    autowire: true
```

Comment:
As quick_access.yml is a new file, we should probably promote autowiring defining it by default

```yaml
services:
  _defaults:
    public: false
    autowire: true
```